app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -182,7 +184,7 @@ def pulp3_host!
         pulp3_uri!.host
       end
 
-      def pulp3_url(path = '/pulp/api/v3/')
+      def pulp3_url(path = '/pulp/api/v3')
```

Comment:
My guess is that you only hit this because you were pointing straight at the pulpcore-api rather than going through apache?  I don't see the issue with the normal proxied-apache setup, but I think the change makes sense :+1: